diff -urN ../libreadline-java-0.8.0.orig/src/native/Makefile ./src/native/Makefile
--- ../libreadline-java-0.8.0.orig/src/native/Makefile	2003-01-07 05:14:35.000000000 -0500
+++ ./src/native/Makefile	2007-07-05 13:54:47.000000000 -0400
@@ -45,9 +45,8 @@
 CFLAGS=-DWIN32=$(WIN32) -D__IBMC__
 else
 INCLUDES          = -I $(JAVAINCLUDE) -I $(JAVANATINC)
-LIBPATH           = -L/usr/lib/termcap 
-JavaReadline_LIBS = -lreadline -ltermcap -lhistory
-JavaEditline_LIBS = -ledit -ltermcap
+JavaReadline_LIBS = -lreadline -ltinfo -lhistory
+JavaEditline_LIBS = -ledit -lncurses
 ifeq (cygwin,$(WIN32))
 JavaGetline_LIBS = -lcygwin
 endif
@@ -97,7 +96,7 @@
 	$(CC) -W1,$(ROOTDIR)/lib$(TG).exp -o $@ \
 		$(OBJECTS) -W1,-e,_readline_init@12
 else
-	$(CC) -shared $(OBJECTS) $(LIBPATH) $($(TG)_LIBS) -o $@
+	$(CC) -shared $(OBJECTS) $($(TG)_LIBS) -o $@
 endif
 endif
 
